<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>StatusResumeFrame</class>
 <widget class="QWidget" name="StatusResumeFrame">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>160</width>
    <height>63</height>
   </rect>
  </property>
  <property name="focusPolicy">
   <enum>Qt::NoFocus</enum>
  </property>
  <property name="contextMenuPolicy">
   <enum>Qt::NoContextMenu</enum>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="autoFillBackground">
   <bool>false</bool>
  </property>
  <property name="styleSheet">
   <string notr="true">#StatusResumeFrame { 
	border: 2px solid grey;
	box-shadow:20px 20px 10px grey;
	border-radius: 5px; 
	background-color:white
}</string>
  </property>
  <widget class="QWidget" name="verticalLayoutWidget">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>160</width>
     <height>62</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout" stretch="0,0,0,0">
    <property name="spacing">
     <number>0</number>
    </property>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout" stretch="0,1">
      <property name="spacing">
       <number>2</number>
      </property>
      <item>
       <widget class="QLabel" name="status_title">
        <property name="text">
         <string>Status: </string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="status_value">
        <property name="text">
         <string>Desc</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_2" stretch="0,1">
      <property name="spacing">
       <number>2</number>
      </property>
      <item>
       <widget class="QLabel" name="start_title">
        <property name="text">
         <string>Start:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="start_value">
        <property name="text">
         <string>Start</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_4" stretch="0,1">
      <property name="spacing">
       <number>2</number>
      </property>
      <item>
       <widget class="QLabel" name="end_title">
        <property name="text">
         <string>End:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="end_value">
        <property name="text">
         <string>End</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_5" stretch="0,1">
      <property name="spacing">
       <number>2</number>
      </property>
      <item>
       <widget class="QLabel" name="duration_title">
        <property name="text">
         <string>Duration:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="duration_value">
        <property name="text">
         <string>Duration</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
